cmd/estuary-shuttle/main.go

```diff
@@ -1532,26 +1532,9 @@ func (d *Shuttle) doPinning(ctx context.Context, op *pinner.PinningOperation, cb
 	dsess := dserv.Session(ctx)
 
 	if err := d.addDatabaseTrackingToContent(ctx, op.ContId, dsess, d.Node.Blockstore, op.Obj, cb); err != nil {
-		// pinning failed, we wont try again. mark pin as dead
```

Comment:
this means that the pinning will still be "true" which in this case will be retried?

Reply:
yes, I thought about uncommenting, for the time being, to help with the pressure. Later in the future, we add support for retries - what do you think?

Reply:
Ah, I remember now, I removed this comment because I already made sure the pinning status is marked false if it fails in some PR in the past. the `onPinStatusUpdate` func is called anything a pin attempt fails from here - https://github.com/application-research/estuary/blob/dev/cmd/estuary-shuttle/main.go#L506

Which marks changes the pinning status to false - so no reties